posthog/api/test/dashboards/test_dashboard.py

```diff
@@ -548,6 +548,43 @@ def test_dashboard_item_layout(self):
 
         self.assertTrue("lg" in first_tile_layouts)
 
+    def test_dashboard_tile_color_can_be_set_for_new_or_existing_tiles(self):
+        dashboard_id, _ = self._create_dashboard({"name": "asdasd", "pinned": True})
+
+        insight_id, _ = self._create_insight(
+            {"filters": {"hello": "test"}, "dashboards": [dashboard_id], "name": "another"}
+        )
+
+        dashboard_json = self._get_dashboard(dashboard_id)
+        tiles = dashboard_json["tiles"]
+        assert len(tiles) == 1
+        tile_id = tiles[0]["id"]
+        # layouts used to live on insights, but moved onto the relation from a dashboard to its insights
+        response = self.client.patch(
+            f"/api/projects/{self.team.id}/dashboards/{dashboard_id}",
+            {
+                "tiles": [
+                    {
+                        "id": tile_id,
+                        "color": "red",
+                        "is_cached": True,  # included to ensure we can update tiles with this readonly property
+                    },
+                    {
```

Comment:
The test only fails when creating a new tile. `is_cached` is ignored on update but causes creates to fail 

Need to expand the test case so it is a valid create-able tile